abicompat.lds

```diff
@@ -0,0 +1,10 @@
+SECTIONS
+{
+  .data : {
```

Comment:
Note to self: it seems `ld` on Focal doesn't like reopening `.data`.
Workaround: change the section to a brand new subsection.

```suggestion
  .data.abi_compat : {
```